mongo/integration/server_selection_prose_test.go

```diff
@@ -0,0 +1,151 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
```

Comment:
```suggestion
// Copyright (C) MongoDB, Inc. 2022-present.
```

Reply:
Is there an official policy on what copyright year to put in new files? Should it be the same value for all files in the driver or the year when the file was added? Either way, seems like a good suggestion.

Reply:
Updated.

Reply:
I'm not sure what our policy is exactly. From what I've gathered if we make a new code file, we should mark it with a copyright beginning with the current year. I suppose that makes intuitive sense, as the copyright should always cover from the point of file creation to present.